UI-MVC/Views/Project/Create.cshtml

```diff
@@ -0,0 +1,94 @@
+@using System.Collections
+@using DAL.Data_Transfer_Objects
+@using Domain.Projects
+@using Domain.Users
+@model UIMVC.Models.ProjectViewModel
+@{
+    ViewBag.Title = "Create";
+    Layout = "_Layout";
+
+    IEnumerable<Platform> platforms = (IEnumerable<Platform>) ViewData["platforms"];
+}
+
+<h2>Create Project</h2>
+
+
+<form asp-controller="Project" asp-action="Create" method="post">
+
+    <div class="form-row">
+        <div class="form-group col-md-12">
+            <label asp-for="Project.Title">Naam</label>
+            <input asp-for="Project.Title" class="form-control" value="Test"/>
+        </div>
+        <div class="form-group col-md-6">
+            <label asp-for="Project.StartDate">StartDate</label>
+            <input type="date" asp-for="Project.StartDate" class="form-control" value="1/1/0001 12:00:00 AM"/>
+        </div>
+        <div class="form-group col-md-6">
+            <label asp-for="Project.EndDate">EndDate</label>
+            <input type="date" asp-for="Project.EndDate" class="form-control" value="1/1/0001 12:00:00 AM"/>
+        </div>
+        <div class="form-group col-md-6">
+            <label asp-for="Project.Goal">Hoofdvraag</label>
+            <input asp-for="Project.Goal" class="form-control" value="Test"/>
+        </div>
+        <div class="form-group col-md-6">
+            <label asp-for="Project.Status">Status</label>
+            <input asp-for="Project.Status" class="form-control" value="Open"/>
+        </div>
+    </div>
+    <div class="form-group">
+        <input type="checkbox" asp-for="Project.Visible"/>
+        <label asp-for="Project.Visible">Visible</label>
+    </div>
+
+
+    <div class="form-group">
+
+        <label asp-for="Project.Platform.Name">Platform Naam</label>
```

Comment:
Als je de button verplaats kan je in de asp-route een platform id meegeven waardoor je geen platform meer 
moet kiezen (wat je nu doet is bad practice want je haalt miss tientallen platformen op ipv 1 idke op te 
sturen)